roles/kubernetes/node/defaults/main.yml

```diff
@@ -93,6 +93,8 @@ kube_cadvisor_port: 0
 # The read-only port for the Kubelet to serve on with no authentication/authorization.
 kube_read_only_port: 0
 
+# sysctl_file_path to add sysctl conf to
+sysctl_file_path: "/etc/sysctl.conf"
```

Comment:
We can't set two separate paths with the same var and expect Ansible to behave. Please set it to ` "/etc/sysctl.d/99-sysctl.conf" `

Reply:
Actually I didn't want to change the current behaviour in roles/kubernetes/node/tasks/main.yml (which is to update /etc/sysctl.conf regarding https://docs.ansible.com/ansible/2.4/sysctl_module.html).
The scope of roles/kubernetes/node/defaults/main.yml is only the role.

Reply:
@mattymo It seems that, on most distribs, /etc/sysctl.d/99-sysctl.conf is a symlink to /etc/sysctl.conf but I don't know for sure the impact of changing the baheviour of this particular role.
My initial purpose of having this variable in defaults was to have a separate file specific to kubespray, so setting it to /etc/sysctl.d/99-sysctl.conf is fine for me.
Let me know.